src/app/Caches/Create/Features/Backups/BackupsCache.tsx

```diff
@@ -0,0 +1,205 @@
+                        <TextInput value={remoteCache} onChange={(val) => { val === "" ? setRemoteCache(undefined!) : setRemoteCache(val) }} aria-label="remote-cache-input" />
+                    </FormGroup>
+                    <FormGroup fieldId='remote-site' isInline>
+                        <MoreInfoTooltip label={t('caches.create.configurations.feature.remote-site')} toolTip={t('caches.create.configurations.feature.remote-site-tooltip')} textComponent={TextVariants.h3} />
+                        <TextInput value={remoteSite} onChange={(val) => { val === "" ? setRemoteSite(undefined!) : setRemoteSite(val) }} aria-label="remote-site-input" />
+                    </FormGroup>
+                </Form>
+            </ExpandableSection>
+        )
+    }
+
+    return (
+        <Card>
+            <CardHeader>
+                <TextContent>
+                    <Text component={TextVariants.h2}>{t('caches.create.configurations.feature.backups')}</Text>
+                    <Text component={TextVariants.p}>{t('caches.create.configurations.feature.backups-description', { brandname: brandname })}</Text>
+                </TextContent>
+            </CardHeader>
+            <CardBody>
+                {formBackupSites()}
+            </CardBody>
+            <CardBody>
+                {sites.map(currentChip => (
+                    <Label
+                        color="blue"
+                        closeBtnAriaLabel="Remove sites"
+                        onEditComplete={(text) => {
+                            editChip(currentChip, text);
+                        }}
+                        isEditable
+                        style={{ marginRight: global_spacer_sm.value }}
+                        key={currentChip.siteName}
+                        onClose={() => deleteChip(currentChip)}>
+                        {currentChip.siteName}
+                    </Label>
+                ))}
+            </CardBody>
+            <CardBody>
+                {typeof sites !== undefined && sites.length > 0 &&
+                    <MoreInfoTooltip label={t('caches.create.configurations.feature.strategy')} toolTip={t('caches.create.configurations.feature.strategy-tooltip')} textComponent={TextVariants.h3} />}
+                {sites.map((currentChip, index) => {
+                    currentChip.siteStrategy = currentChip.siteStrategy || 'ASYNC';
+                    return (
+                        <FormGroup key={index}
+                            fieldId='strategy'
+                            isInline
+                            style={{ paddingTop: global_spacer_sm.value }}
+                        >
+                            <TextContent>
+                                <Text component={TextVariants.h3}>{currentChip.siteName}</Text>
+                            </TextContent>
+                            <Radio
+                                name={index.toString()}
```

Comment:
@karesti choosing a radio option when multiple sites are there, is not working properly, please check

Reply:
I'm working on this today

Reply:
As the options are only async and sync and need to select only one, radio options will be good,
The issue mentioned above got solved, please check.